modules/openthread/platform/openthread-core-zephyr-config.h

```diff
@@ -510,4 +510,14 @@
 #define OPENTHREAD_CONFIG_MLE_CHILD_TIMEOUT_DEFAULT CONFIG_OPENTHREAD_MLE_CHILD_TIMEOUT
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_PSA_ITS_NVM_OFFSET
+ *
+ * NVM offset while using key refs.
```

Comment:
I know it is similarly described in OT.. but maybe:
```suggestion
 * Offset of IDs for OpenThread keys stored in the PSA ITS.
```